iptc/xtables.py

```diff
@@ -798,10 +798,15 @@ class XTablesError(Exception):
 _optind = ct.c_long.in_dll(_libc, "optind")
 _optarg = ct.c_char_p.in_dll(_libc, "optarg")
 
-_lib_xtables, xtables_version = find_library("xtables")
+xtables_version = os.getenv("PYTHON_IPTABLES_XTABLES_VERSION")
+if xtables_version:
+    _searchlib = "libxtables.so.%s" % (xtables_version,)
+else:
+    _searchlib = "xtables"
+print(_searchlib)
```

Comment:
Can you remove this print statement? I assume it's only there for debugging purposes.

Reply:
Ah, indeed. Fixed now.